### ISSUE TITLE
[Issue]: Dashboard and top navigation - Fix color contrast and structure issues found by axe-dev

### ISSUE THREAD

**Issue by lsr-explore** [usdigitalresponse/usdr-gost#3755]
### Why is this issue important?

If the color contrast is not sufficient, low vision and color blind users may have difficulty seeing the elements on the page

Incomplete structure can impact the navigability of the site for customers who use screen readers

Findings from [axe devTools chrome extension](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd)

## Color Contrast ratio threshold
WCAG 1.43 AA
![image](https://github.com/user-attachments/assets/07205510-16a9-4633-b988-4c0327dda662)
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

All elements that failed color contrast shown from [Microsoft's Accessibility Insights Chrome Extension](https://chromewebstore.google.com/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni) 
![image](https://github.com/user-attachments/assets/7faec909-5b7c-47f7-b300-284e80634dd9)

---------------
## Structure

![image](https://github.com/user-attachments/assets/b6ec49b7-23e8-4d23-b926-c827449cff39)

### Current State

When user visits ARPA reporter home page - https://staging.grants.usdr.dev/arpa_reporter/

There are accessibility issues that may impact customer's ability to use the site.

### Expected State

When user visits ARPA reporter home page - https://staging.grants.usdr.dev/arpa_reporter/

No accessibility issues

## Color Contrast
* Use USDR blue used in Grants Finder for the blue links
* Darken the light gray text
* Use USDR blue for the blue buttons
* Identify a darker version of the cyan color for the "Send Audit Report by Email" button

## Structure
* Add main landmark for the main content
* Add an H1 heading element

**Comment by ClaireValdivia** [comment]
This all looks great in staging!
